examples/evaluation_and_profiling/swe_bench/src/nat_swe_bench/predictors/predict_iterative/tools/register.py

```
@@ -0,0 +1,60 @@
+        args = json.loads(args_str)
+        operation = args.get('operation')
+
+        if operation == "setup":
+            context = await repo_manager.setup_repository(args['repo_url'], args['base_commit'])
+            return str(context.repo_path)
+
+        if operation == "cleanup":
+            await repo_manager.cleanup()
+            return "Cleanup complete"
+
+        raise ValueError(f"Unknown operation: {operation}")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)